content/de/how-it-works.md

```diff
@@ -8,57 +8,59 @@ lastmod: 2019-10-18
 
 {{< lastmod >}}
 
-Das Ziel von Let's Encrypt und des [ACME-Protokolls](https://tools.ietf.org/html/rfc8555) besteht darin, die Einrichtung eines HTTPS-Servers zu ermöglichen, der automatisch ein vertrauenswürdiges Browserzertifikat ohne menschliches Eingreifen erhält. Dies wird durch Ausführen eines Zertifikatsverwaltungsagenten auf dem Webserver erreicht.
+Das Ziel von Let's&nbsp;Encrypt und des [ACME-Protokolls](https://tools.ietf.org/html/rfc8555) besteht darin, die Einrichtung eines HTTPS-Servers zu ermöglichen, der automatisch ein vertrauenswürdiges Browserzertifikat ohne menschliches Eingreifen erhält.  Dies wird durch Ausführen eines Zertifikatsverwaltungsagenten auf dem Webserver erreicht.
 
-Um zu verstehen, wie die Technologie funktioniert, gehen wir die Einrichtung von `https://example.com/` mit einem Zertifikatsverwaltungsagenten durch, der Let's Encrypt unterstützt.
-
-Dieser Prozess umfasst zwei Schritte. Zunächst weist der Agent der Zertifizierungsstelle nach, dass der Webserver eine Domain kontrolliert. Anschliessend kann der Agent Zertifikate für diese Domain anfordern, erneuern und widerrufen.
+Um zu verstehen, wie die Technologie funktioniert, gehen wir die Einrichtung von `https://example.com/` mit einem Zertifikatsverwaltungsagenten durch, der Let's&nbsp;Encrypt unterstützt.
 
+Dieser Prozess umfasst zwei Schritte.  Zunächst weist der Agent der Zertifizierungsstelle nach, dass der Webserver eine Domain kontrolliert.  Anschließend kann der Agent Zertifikate für diese Domain anfordern, erneuern und widerrufen.
```

Comment:
Dieser Prozess umfasst zwei Schritte. Zunächst weist der Agent der Zertifizierungsstelle nach, dass der Webserver eine Domain kontrolliert. Anschließend kann der Agent Zertifikate für diese Domain anfordern, erneuern und widerrufen.

Reply:
@tdelmas It seems like there are always two spaces between individual strings in this file. This is not present when rendered to HTML, so I will leave it as-is.

Reply:
(and I can't change it on Crowdin)

Reply:
Ok, I removed the double spaces.